api/migrations/0011_contactus_feedback.py

```diff
@@ -0,0 +1,58 @@
+# Generated by Django 3.1.4 on 2021-04-26 22:01
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("api", "0010_auto_20210402_2245"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="ContactUs",
+            fields=[
+                (
+                    "id",
+                    models.AutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("email", models.CharField(blank=True, max_length=50)),
+                ("name", models.CharField(blank=True, max_length=50)),
+                ("message", models.CharField(max_length=2000, null=True)),
+            ],
+        ),
+        migrations.CreateModel(
+            name="Feedback",
+            fields=[
+                (
+                    "id",
+                    models.AutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                (
+                    "reason",
+                    models.CharField(
+                        choices=[
+                            ("W", "Website"),
+                            ("Pp", "Penpal"),
+                            ("P", "Performance"),
+                        ],
+                        max_length=2,
+                        null=True,
+                    ),
+                ),
+                ("message", models.CharField(blank=True, max_length=2000)),
```

Comment:
Just curious why this limit on message length.

Reply:
@AyalaCnaan When a user sends their feedback, the maximum length the database will hold is 2000.

Reply:
For technical reasons, [the `CharField` requires a max length be specified](https://docs.djangoproject.com/en/3.2/ref/models/fields/#django.db.models.CharField.max_length). A [`TextField`](https://docs.djangoproject.com/en/3.2/ref/models/fields/#textfield) does not have this restriction and would probably be better suited for something like the feedback message.

As for why 2000 was specifically chosen... :man_shrugging: